website/community/committers/release.md

```diff
@@ -341,15 +341,12 @@ Checklist for reference:
 [ ] All source files have ASF headers
 [ ] Can compile from source
 
-More detailed checklist please refer to:
-https://github.com/apache/opendal/tree/main/scripts
+Use our verify.py to assist in the verify process:
 
-To compile from source, please refer to:
-https://github.com/apache/opendal/blob/main/CONTRIBUTING.md
-
-Here is a Python script in release to help you verify the release candidate:
-
-./scripts/verify.py
+svn co https://dist.apache.org/repos/dist/dev/opendal/${release_version}/ opendal-dev
+cd opendal-dev
+curl -sSL https://github.com/apache/opendal/raw/${release_version}/scripts/verify.py -o verify.py
```

Comment:
As we defined: `The ${release_version} here should be like 0.36.0-rc.1`. But here we need `v0.36.0-rc.1` not `0.36.0-rc.1`

https://raw.githubusercontent.com/apache/opendal/0.45.1-rc.4/scripts/verify.py
https://raw.githubusercontent.com/apache/opendal/v0.45.1-rc.4/scripts/verify.py

Reply:
Fixed!